### PR TITLE
User has the option to enter SMILE data as a csv

### PR DESCRIPTION
**Pull request by sarah-witt**
Added the option for a user to enter SMILE data as a csv. 

This also fixes a small bug in the pains filter and filters the compounds naively (just removes the ones that failed).

### CODE REVIEW

File: app/routes.py [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
+        good_smiles = filter_smiles(pains.get_smiles(inputs))
 
-  get_smiles_json(tanimoto_smiles)
+        tanimoto_smiles = clustering.add_tanimoto_coefficients(good_smiles)
 
+        get_smiles_json(tanimoto_smiles)
 
-  return render_template('cluster.html', title='Cheminformatic Analysis', bad_smiles=bad_smiles, tanimoto_smiles=tanimoto_smile_strings)
\ No newline at end of file
+        return render_template('cluster.html', title='Cheminformatic Analysis')
```
Comment (aislinblack) [inline app/routes.py:31]: Are we planning on changing where this code goes later? Because it seems kind of odd to go straight from input to cluster.

**Review by aislinblack — APPROVED** [review]
Looks great!

File: app/routes.py (hunk 1 above)
Comment (sarah-witt) [reply app/routes.py:31]: Yeah, probably! If we can add the manual PAINS selection then that would go before clustering.

Comment (aislinblack) [reply app/routes.py:31]: If you merge it now, then I can fit those next step changes into my branch.

Comment (sarah-witt) [reply app/routes.py:31]: awesome! I'll merge

**Pull request merged**